### ISSUE TITLE
Long Form Note [NIP-23] Positive Interoperability Tracker

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#97]
Spec: https://github.com/nostr-protocol/nips/blob/master/23.md

**app**|**View**|**Publish**|**comment**
-----|-----|-----|-----
https://untype.app/||✅|can cross-post to ghost| 
comet.md |❌ | ✅ ||
[Texstr](https://www.texstr.org/)|✅  | ✅ ||[supports LaTex](https://github.com/nostrability/nostrability/issues/146#issuecomment-3053102700)
notestack.com | ✅ | ✅ | |
yakihonne|✅|✅|| web and mobile
habla.news|✅|✅|| 
highlighter|✅|✅|| 
Primal|✅ |✅|[Primal article overview](https://njump.me/nevent1qqsdxku9htc2pwfdcdqw9vqt7k2ukqvr9g852y6qlsgccntw6eldwpspzemhxue69uhhyetvv9ujuurjd9kkzmpwdejhgqghwaehxw309aex2mrp0yhxxatjwfjkuapwveukjqg5waehxw309aex2mrp0yhxummnw3ezucn8qyxhwumn8ghj7mn0wvhxcmmv0atfdj)|
zap.cooking|✅*|✅*|*TBC |
[pareto.space](https://pareto.space/read)|✅|*|*TBC |https://github.com/twenty-eighty/pareto-client
njump|✅|❌|lightweight web app can be used to verify an article was published successfully| 
damus|✅|❌|publishing not planned on iOS| 
amethyst|✅|❌|| 
coracle|✅|❌||  
gossip|✅*|❌|*Rendered but as plaintext|
nostur | ✅ | ❌ | iOS only mobile client |
nostr.band|❌|❌|| 
Longform.space|✅||nip-07 log in required
[makimono/lumilumi](https://makimono.lumilumi.app/)|✅|✅|
decentnewsroom.com|✅|✅|
jumble|✅||`LongFormArticle` component, kind:30023 rendering
snort|✅|❌||na-gro audit: renders kind 30023 via @snort/system
lume|✅|❌||na-gro audit: desktop client, NIP-23 rendering
nostria|✅|✅||na-gro audit: 48 NIPs, includes NIP-23 longform
Divine|✅|❌||divine audit: kind 30023 support in mobile SDK

**Comment by alltheseas** [comment]
@brugeman you planning on adding nip-23 long form note to nostr.band?

**Comment by alltheseas** [comment]
https://njump.me/naddr1qvzqqqr4gupzq96n3hp2vfmf6z2y8uvvxl97xk86kkalnqghx4p25lzl79c76a7yqy88wumn8ghj7mn0wvhxcmmv9uq3xamnwvaz7tmsw4e8qmr9wpskwtn9wvhszxnhwden5te0wpuhyctdd9jzuenfv96x5ctx9e3k7mf0qywhwumn8ghj7mr4de3kscn00qh8xctwv3mkjcmg9enxzund9uq3wamnwvaz7tmjv4kxz7fwdehhxtnnda3kjctv9uq3zamnwvaz7te3xsczue3h0ghxjme0qy2hwumn8ghj7un9d3shjtnyv9kh2uewd9hj7qgnwaehxw309amk7apww468smewdahx2tcpr4mhxue69uhkummnw3ez6vp39eukz6mfdphkumn99e3k7mf0qy2hwumn8ghj7mn0wd68ytn00p68ytnyv4mz7qq4fycnsvfdg43kku35gyuy2jtcwdexjazrxqdralzu 

Noticed that 
1) njump supports long form very well 
2) hashtag I published in the longform note above (#eulag) does not appear in search in damus, nostr.band, coracle

**Comment by alltheseas** [comment]
@mbrat1 I don't see long notes on primal web - are you planning support?

**Comment by alltheseas** [comment]
@mmalmi I don't see the long form note on Iris in my posts - do you plan to add view long form note support?

**Comment by fiatjaf** [comment]
I honestly think microblogging clients shouldn't support this. It's fine if they do, but we shouldn't be actively pushing for support.

**Comment by jb55** [comment]
> I honestly think microblogging clients shouldn't support this. It's fine if they do, but we shouldn't be actively pushing for support.

agree, it's not something that is super important to interoperate on. I don't have any plans to do longform publishing in damus ios. A standalone "write room" app makes more sense for that.

**Comment by alltheseas** [comment]
@fiatjaf as added a bounty for adding NIP-23 to RSS readers

https://njump.me/nevent1qvzqqqqqqypzqwlsccluhy6xxsr6l9a9uhhxf75g85g8a709tprjcn4e42h053vaqqsqqqqns8wf9ukah2ynpzdxgur30dwzn6er4msrqnge35pzhdqn0ns7t9een

**Comment by alltheseas** [comment]
added bounty tracker here: https://github.com/nostrability/nostrability/issues/134

**Comment by alltheseas** [comment]
comet.md dev advised added support for publishing articles. 

Was not successful in publishing. Will try again next month.

**Comment by alltheseas** [comment]
Notestack view and publish confirmed. Added

**Comment by Sebastix** [comment]
I use [Nostur](https://nostur.com) a lot for reading long-form content.

**Comment by alltheseas** [comment]
https://njump.me/nevent1qqsxe8a40fw4dsfrz62urzr2u266vejnu87prn3yd8za79xgc77yhqqpz3mhxue69uhhyetvv9ujuerpd46hxtnfduqs6amnwvaz7tmwdaejumr0dsq3vamnwvaz7tmjv4kxz7fwwpexjmtpdshxuet5akm9xj

https://untype.app/

Can publish

**Comment by alltheseas** [comment]
Primal launched article editor

https://njump.me/nevent1qqsdxku9htc2pwfdcdqw9vqt7k2ukqvr9g852y6qlsgccntw6eldwpspzemhxue69uhhyetvv9ujuurjd9kkzmpwdejhgqghwaehxw309aex2mrp0yhxxatjwfjkuapwveukjqg5waehxw309aex2mrp0yhxummnw3ezucn8qyxhwumn8ghj7mn0wvhxcmmv0atfdj

> Hey Nostr, the Primal Article Editor is here! 

> Our goal was to build a feature-rich, rock-solid long form note editor that interoperates perfectly with other Nostr long form apps like Highlighter, Habla, and Yakihonne. 

> It’s a complex product with a lot of moving parts—not easy to nail on the first try. We think we got pretty close. Try it out and let us know what you think. We’ll keep improving it until it’s tip-top. 🫡 https://primal.net/myarticles

**Comment by alltheseas** [comment]
adding pareto.space

![Image](https://github.com/user-attachments/assets/4f7aa8ce-ce0d-462c-8de7-16634326bc83)

https://pareto.space/a/naddr1qqjk67fdwajk26mn955rzwptxyujjttfdckhyetkd9jhwtfjxqhnqdf0xgcrydgpp4mhxue69uhkummn9ekx7mqzyzamthdqu92k09ulq4p5q77uyqeadu9mkv8hy5f2nqw0mvhsncn5wqcyqqq823cgqde22

**Comment by alltheseas** [comment]
added longform.space

also learned that @yakihonne supports publication via mobile

**Comment by alltheseas** [comment]
https://zap-tracker.netlify.app/ advised they will add NIP-23 support

**Comment by alltheseas** [comment]
Notecrumbs tenders longform:

https://damus.io/nevent1qqswpquv0vln2xeq6qlvslldhtr3yut7e6gpt0kypl40j033mtqlzuspp4mhxue69uhkummn9ekx7mqzyzm7669svt0xkjsju50a22zurc0qa589z2xd4yatzx6p2z64a5e0cqcyqqq823canvy4l

Https//github.com/damus-io/notecrumbs

![image](https://github.com/user-attachments/assets/e8278570-77e8-4ad0-8df4-ed5dd2579a59)

![image](https://github.com/user-attachments/assets/aae7629b-2040-4494-b3a2-2dbd65d17f1d)
